content/10.zk-stack/20.running-a-zk-chain/80.validium.md

```diff
@@ -181,23 +181,155 @@ updates/optimizations or completely changing the DA verification logic, e.g. due
 <details>
 <summary>How to configure EigenDA client?</summary>
 
-1. Add the following block to the `general.yaml`, values used are only an example:
+## Common changes
+
+Changes needed both for local and mainnet/testnet setup.
+
+1. Add `da_client` to `etc/env/file_based/general.yaml`:
```

Comment:
I'm not sure about `etc/env/file_based` as those are the base for all the chains, I'd prefer we use a generic pattern to point to the specific chain config